tasks/test-project/tui-tasks.js

```diff
@@ -406,38 +408,76 @@ async function apiTasks(outputPath, { linkWithLatestFwBuild }) {
       },
     })
 
-    // At an earlier step we run `yarn rwfw project:copy` which gives us
-    // auth-dbauth-setup@3.2.0 currently. We need that version to be a canary
-    // version for auth-dbauth-api and auth-dbauth-web package installations
-    // to work. So we update the package.json to make the setup use the latest
-    // canary version for the api and web sides
+    // We want to use the latest version of the auth-dbauth-{setup,api,web}
+    // packages. But they're not published yet. So let's package them up as
+    // tarballs and install them using that by setting yarn resolutions
 
-    const { stdout } = await exec(
-      `yarn npm info @redwoodjs/auth-dbauth-setup --fields versions --json`,
-      [],
-      execaOptions
+    const setupPkg = path.join(
+      RW_FRAMEWORK_PATH,
+      'packages',
+      'auth-providers',
+      'dbAuth',
+      'setup'
+    )
+    const apiPkg = path.join(
+      RW_FRAMEWORK_PATH,
+      'packages',
+      'auth-providers',
+      'dbAuth',
+      'api'
     )
+    const webPkg = path.join(
+      RW_FRAMEWORK_PATH,
+      'packages',
+      'auth-providers',
+      'dbAuth',
+      'web'
+    )
+
+    await exec('yarn build:pack', [], getExecaOptions(setupPkg))
+    await exec('yarn build:pack', [], getExecaOptions(apiPkg))
+    await exec('yarn build:pack', [], getExecaOptions(webPkg))
 
-    const latestCanaryVersion = JSON.parse(stdout)
-      .versions.filter((version) => version.includes('canary'))
-      .at(-1)
+    const setupTgz = path.join(setupPkg, 'redwoodjs-auth-dbauth-setup.tgz')
+    const apiTgz = path.join(apiPkg, 'redwoodjs-auth-dbauth-api.tgz')
+    const webTgz = path.join(webPkg, 'redwoodjs-auth-dbauth-web.tgz')
 
-    const dbAuthSetupPath = path.join(
+    const setupTgzDest = path.join(
+      outputPath,
+      'node_modules',
```

Comment:
Ok if we move the tarballs somewhere other than `node_modules`? Like just in the project's root directory? Or even in `.redwood`? Yarn will unpack the tarballs into `node_modules`. I'm not sure if it actually matters or not but I'd rather treat `node_modules` as Yarn's domain

Reply:
Copied to root, and added code to make sure we delete them to keep things clean